src/Akka.Persistence.Sql.Linq2Db/persistence.conf

```diff
@@ -240,86 +269,11 @@
-          auto-init = true
-
-          #if true, a warning will be logged
-          #if auto-init of tables fails.
-          #set to false if you don't want this warning logged
-          #perhaps if running CI tests or similar.
-          warn-on-auto-init-fail = true
-
-          table-name = "journal"
-          metadata-table-name = "journal_metadata"
-
-          #If you want to specify a schema for your tables, you can do so here.
-          schema-name = null
-
-
-          column-names {
-            "ordering" = "ordering"
-            "deleted" = "deleted"
-            "persistenceId" = "persistence_id"
-            "sequenceNumber" = "sequence_number"
-            "created" = "created"
-            "tags" = "tags"
-            "message" = "message"
-            "identifier" = "identifier"
-            "manifest" = "manifest"
-          }
-          sqlserver-compat-column-names {
-            "ordering" = "ordering"
-            "deleted" = "isdeleted"
-            "persistenceId" = "persistenceId"
-            "sequenceNumber" = "sequenceNr"
-            "created" = "Timestamp"
-            "tags" = "tags"
-            "message" = "payload"
-            "identifier" = "serializerid"
-            "manifest" = "manifest"
-          }
-          sqlite-compat-column-names {
-            "ordering" = "ordering"
-            "deleted" = "is_deleted"
-            "persistenceId" = "persistence_Id"
-            "sequenceNumber" = "sequence_nr"
-            "created" = "Timestamp"
-            "tags" = "tags"
-            "message" = "payload"
-            "identifier" = "serializer_id"
-            "manifest" = "manifest"
-          }
-          postgres-compat-column-names {
-            "ordering" = "ordering"
-            "deleted" = "is_deleted"
-            "persistenceId" = "persistence_id"
-            "sequenceNumber" = "sequence_nr"
-            "created" = "created_at"
-            "tags" = "tags"
-            "message" = "payload"
-            "identifier" = "serializer_id"
-            "manifest" = "manifest"
-          }
-          metadata-column-names {
-            "persistenceId" = "persistenceId"
-            "sequenceNumber" = "sequenceNr"
-          }
-          sqlserver-compat-metadata-column-names {
-            "persistenceId" = "persistenceId"
-            "sequenceNumber" = "sequenceNr"
-          }
-          sqlite-compat-metadata-column-names {
-            "persistenceId" = "persistence_Id"
-            "sequenceNumber" = "sequence_nr"
-          }
-          postgres-compat-metadata-column-names {
-            "persistenceId" = "persistence_id"
-            "sequenceNumber" = "sequence_nr"
-          }
-        }
+        tag-separator = ","
+      
+        dao = "Akka.Persistence.Sql.Linq2Db.Journal.DAO.ByteArrayJournalDao, Akka.Persistence.Sql.Linq2Db"
+      
+        tables.journal = ${akka.persistence.journal.linq2db.tables.journal}
```

Comment:
Since the query and journal `tables.journal` is the same, we can replace the whole block with a substitution